man/coredump.conf.xml

```diff
@@ -86,6 +86,18 @@
         <listitem><para>The maximum size in bytes of a core
         which will be processed. Core dumps exceeding this size
         may be stored, but the backtrace will not be generated.
+        Like other sizes in this same config file, SI suffixes
+        are allowed (B, K, M, G, T, P, and E.) For example, 
+        1G may be used in place of 1073741824 bytes. Decimals
+        are allowed.
```

Comment:
they *aren't* SI suffixes here, they are a corruption of the IEC suffixes. SI means to the base of 1000, but we use 1024 here. IEC uses 1024, but then uses these weird "Kibi", "Mibi" names, which outside of standards nerd circles noone actually uses. We use 1024 as base here, but just call it K, M, G, T, …, i.e. we do what is actually commonly done, but document it.

I'd really not get lost in IEC vs SI here. Just say "The usual suffixes K, M, G, T, … to the base 1024 are supported.". That's brief, avoids the politics, and says exactly what you need to know and is what we usually say elsewhere in the docs.

(Across the codebase we try to keep this short, always use K/M/G/T as suffixes, but depending on context it means 1000 sometimes, and 1024 sometimes, following how the suffix is usually used in the relevant area. For example network device speeds are usually measured in bits per second to the base of 1000, while disk space usually is measured to the base 1024, hence we do exactly what is common in that area, always with the same suffix, but documenting which one it is)

Reply:
(I mean, we might want to add a separate man page explaining all this in all the detail, the same way how we have a man page of its own to explain how we denote time spans and time stamps in systemd.time(5), but I guess that's material for another PR — work on that would be very welcome though)

Reply:
I've corralled the verbosity hounds and hopefully this latest commit strikes a good balance.